lib/util/common.js

```diff
@@ -68,8 +68,14 @@ function resolveLocalRef (jsonSchema, externalSchemas) {
   if (typeof jsonSchema.type !== 'undefined' && typeof jsonSchema.properties !== 'undefined') {
     // for the shorthand querystring/params/headers declaration
     const propertiesMap = Object.keys(jsonSchema.properties).reduce((acc, headers) => {
-      const required = (Array.isArray(jsonSchema.required) && jsonSchema.required.indexOf(headers) >= 0) || false
-      const newProps = Object.assign({}, jsonSchema.properties[headers], { required })
+      const rewriteProps = {}
```

Comment:
`common.js` is used in both `openapi` and `swagger`. It may cause problem when people mixing both behavior.

Reply:
All tests passed.

Reply:
Yes, it is. but what I means is the same passed variables for `swagger`. It should throw an error when validating because it do not handle the `rawRequired`, etc. It should be a missing test case.

Reply:
Got it. I'll think something

Reply:
@climba03003 if I throw an exception, the user will not see it without logger. I have not found any exceptions in the project. Do you really want throw an exception? Or I can just delete `rawRequired` and `consume` fields in swagger

Reply:
It should be visible when calling `fastify.swagger`, if they follow the document.
```
fastify.ready(err => {
  if (err) throw err
  fastify.swagger() // it throw in here
})
```
or they cannot get a proper `json` when access `/documentation/json`.

Reply:
Ready. `plainJsonObjectToSwagger2` not throw exception